### PR TITLE
Re-organize volunteer information

### PR DESCRIPTION
**Pull request by kirang89**
This PR reorganizes volunteer information.

### CODE REVIEW

**Comment by kirang89** [comment]
This is an initial draft. I'll add more information tomorrow.

File: team.html [hunk 1]
```diff
@@ -232,7 +232,10 @@ <h2 class="text-center">
             <div class="clearfix"></div>
             <br>
             <br>
-            <h3>Core Volunteers</h3>
+            <h3>Core Volunteers & Staff Information</h3>
```
Comment (vnbang2003) [inline team.html]: Let make it just Staff information instead of Core Volunteers & Staff Informaiton

**Comment by kracekumar** [comment]
@kirang89 I'm waiting!

**Comment by kirang89** [comment]
Sorry for the delay. Do review and comment.

**Pull request merged**